programs/ziti-edge-tunnel/package/ziti-edge-tunnel.sh.in

```diff
@@ -0,0 +1,32 @@
+#!@BOURNE_SHELL_BIN@
```

Comment:
@sabedevops If possible I'd like to define the bourne shebang during install based upon the possible paths to `sh` executable.